### ISSUE TITLE
Signup: One step flows: Prevent the language picker from being displayed on the success screen

### ISSUE THREAD

**Issue by fabianapsimoes** [Automattic/wp-calypso#13557]
Reported by @ranh:

We should prevent the language picker from being displayed on the success screen. @drewblaisdell  initially thought that this was a signup-wide issue, but was not able to reproduce this through the main signup flow.

**Comment by pento** [comment]
I just ran through the social signup flow, and didn't see a language picker.

Can you confirm how to reproduce this?

**Comment by fabianapsimoes** [comment]
@pento I was able to reproduce this by following these steps:

- Visit calypso.live/start/social, note that the language picker is displayed there (if not, you might want to add another language to your browser).
- Click Continue with Google and complete the authentication process
- Once the pop-up is closed, you'll be redirected to the processing screen. The language picker is still displayed there:

![screenshot-calypso live-2017-05-12-12-13-53](https://cloud.githubusercontent.com/assets/1248436/25994241/74ac9fbc-370d-11e7-9d1a-612e642ef39b.png)

With further testing, I noticed that this seems to be a general issue with one-step flows. I get the same results when I go through the account creation flow at `/start/account`. I'm updating the title of this issue to reflect that.

@pento if you think this will be an interesting issue for you from a learning point of view, go ahead with it. Otherwise, I'd say it's pretty low priority considering that when social signup is added to the main flow, we'll have a flow with more than one step.
